plugins/modules/cloudformation_stack_set.py

```diff
@@ -83,50 +88,50 @@
     type: int
   capabilities:
     description:
-    - Capabilities allow stacks to create and modify IAM resources, which may include adding users or roles.
-    - Currently the only available values are 'CAPABILITY_IAM' and 'CAPABILITY_NAMED_IAM'. Either or both may be provided.
-    - >
-        The following resources require that one or both of these parameters is specified: AWS::IAM::AccessKey,
-        AWS::IAM::Group, AWS::IAM::InstanceProfile, AWS::IAM::Policy, AWS::IAM::Role, AWS::IAM::User, AWS::IAM::UserToGroupAddition
+      - Capabilities allow stacks to create and modify IAM resources, which may include adding users or roles.
+      - Currently the only available values are 'CAPABILITY_IAM' and 'CAPABILITY_NAMED_IAM'. Either or both may be provided.
+      - >
+          The following resources require that one or both of these parameters is specified: AWS::IAM::AccessKey,
+          AWS::IAM::Group, AWS::IAM::InstanceProfile, AWS::IAM::Policy, AWS::IAM::Role, AWS::IAM::User, AWS::IAM::UserToGroupAddition
     type: list
     elements: str
     choices:
-    - 'CAPABILITY_IAM'
-    - 'CAPABILITY_NAMED_IAM'
+      - 'CAPABILITY_IAM'
+      - 'CAPABILITY_NAMED_IAM'
   regions:
     description:
-    - A list of AWS regions to create instances of a stack in. The I(region) parameter chooses where the Stack Set is created, and I(regions)
-      specifies the region for stack instances.
-    - At least one region must be specified to create a stack set. On updates, if fewer regions are specified only the specified regions will
+      - A list of AWS regions to create instances of a stack in. The I(region) parameter chooses where the Stack Set is created, and I(regions)
+        specifies the region for stack instances.
+      - At least one region must be specified to create a stack set. On updates, if fewer regions are specified only the specified regions will
       have their stack instances updated.
```

Comment:
Many of the sanity test errors are because you forgot to indent this line by two spaces. This makes the `DOCUMENTATION` invalid YAML.

Reply:
@felixfontein I think this module predates the YAML linting check. I have fixed the YAML indentation for the whole documentation block

Reply:
The module had valid YAML indentation before, and passed linting, before you started adjusting indentation.

Reply:
All nested list items were at the same level as their parent, although the tests were passing, the indentation was not correct.

Reply:
That indentation is correct as well. It's valid YAML. That's all that counts.

Just because the default config of yamllint doesn't like it doesn't mean it's not correct.

Reply:
Fair, I should have said "inconsistent" instead of "not correct". Inconsistent because all other other modules use indentation for their documentation. The official YAML spec. also uses two spaces to indent child items (regardless of their types).

Reply:
Yeah, they definitely were not consistent :) About the YAML spec: do you mean the examples in it?

Reply:
Yeah that's what I meant. You're correct though, spaces are defined as a convention or "presentation detail" in the specs ([6.1 Identation Spaces](https://yaml.org/spec/1.2.2/#61-indentation-spaces))